README.md

```diff
@@ -2,7 +2,7 @@
 <img width="631" src="https://www.telerik.com/kendo-angular-ui/npm-banner.svg">
 </a>
 
-## Native UI and Data Visualization Components for Angular
+## Common pieces used for Kendo Angular UI e2e/examples
```

Comment:
Might need a better description here. It's not clear what "pieces" go to which "examples".

Reply:
Pieces is like place holder for 'directives', 'services', 'component' and all other Angular reusable stuff